src/ZipFile.jl

```diff
@@ -423,7 +423,9 @@ end
 # Read data into a. Throws EOFError if a cannot be filled in completely.
 function read(f::ReadableFile, a::Array{T}) where T
     if !isbitstype(T)
-        return invoke(read, Tuple{IO,Array}, f, a)
+        # may need to wrap in invoke() if this package is refactored to overload read!
+        # return invoke(read!, Tuple{IO,Array}, f, a)
+	return read!(f, a)
```

Comment:
This line is indented with a tab. We use 4 spaces for each indentation level.

Reply:
A tab! I must submit my editor to a harsh dressing-down.  May I ask how you check this, in hope of catching it in other cases?

Reply:
I could see the lines are not aligned in chromium, and then if I select the space and see that the selection is wider than how much my mouse moved, it's a tab. Github should really do a better job of displaying space diffs.

Your editor might have a setting to make spaces visible. For example in vim, you can do `:set list`.